daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -50,47 +24,99 @@ Parameter | Description
 `workflowName` | Identify the workflow type
 `instanceId` | Unique value created for each run of a specific workflow
 
+### Request Contents
 
-### Headers
+In the request you can pass along headers:
```

Comment:
We don't accept any specific headers. Did you mean to say something about the workflow input here?

Reply:
I guess so, I forgot 🤷‍♂️

What about this?

```suggestion
In the request you can pass along relevant input information that will be passed to the workflow:
```

Reply:
LGTM!